Notebooks/ReadMe.md

```diff
@@ -0,0 +1,26 @@
+# Project Mu VS Code Notebooks
+
+These notebooks summarize Project Mu information across all of the Project Mu repos.
+
+## How to Use
+
+1. Install [Visual Studio Code (VS Code)](https://code.visualstudio.com/)
+2. Install the `GitHub Issue Notebooks` VS Code extension
+   - Extension ID: ms-vscode.vscode-github-issue-notebooks
+   - [Marketplace link](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-github-issue-notebooks)
+3. Open a notebook file (e.g. `PullRequests.github-issues`)
+4. Click `Run All` at the top of the  file to run all the queries
+
+## How to View Dashboard in Web Browser
+
+Since VS Code can run in your Web browser, you can treat this dashboard as a Web page rather than a file that you open
+locally.
+
+To view the file in the Web version of VS Code, simply open the file in GitHub and replace `github.com` with
+`github.dev` in the URL.
+
+- [Project Mu Pull Request Dashboard](https://github.dev/microsoft/mu_devops/blob/main/Notebooks/PullRequests.github-issues)

```

Comment:
Note that these links won't work until this PR is merged. Here's the equivalent links that work while this PR is active:

- [Project Mu Pull Request Dashboard](https://github.dev/makubacki/mu_devops/blob/add_mu_vscode_github_issues_notebooks/Notebooks/PullRequests.github-issues)
- [Project Mu Personal Issue & Pull Request Dashboard](https://github.dev/makubacki/mu_devops/blob/add_mu_vscode_github_issues_notebooks/Notebooks/MyPullRequests.github-issues)